blocks/init/src/Blocks/components/heading/manifest.json

```diff
@@ -41,11 +41,11 @@
 	"options": {
 		"headingSize": [
 			{
-				"label": "Default (30px)",
+				"label": "Default (1.5em)",
 				"value": "default"
 			},
 			{
-				"label": "Big (50px)",
+				"label": "Big (2.5em)",
```

Comment:
Maybe we should remove units from the labels? Pixels kinda made sense to devs (more traditional), but to clients, the ems won't make any sense.

Reply:
Agreed. Non-dev people would very likely be very confused with ems. We should either leave the pixel values or just remove the units altogether.

Reply:
I will fix this in another PR